### ISSUE TITLE
Add degree to Referee entity

### ISSUE THREAD

**Issue by Tonypsilon** [Tonypsilon/bmm.backend#5]
A referee should have a degree to determine how many matches he can conduct in parallel.

- [ ] Enum RefereeDegree is created with instances "VSR", "RSR", "NSR", "FA", "IA".
- [ ] Referee entity gets a column degree that stores the degree.
- [ ] Database changelog yaml as well as RefereeData, RefereeCreationData, RefereeExternalData and the RefereeService and its test are updated accordingly.

**Comment by Tonypsilon** [comment]
Actually determining if a referee has conducted only as many matches as the degree permits is not part of this issue.
